compose/config/config.py

```diff
@@ -171,6 +171,18 @@ def find_candidates_in_parent_dirs(filenames, path):
     return (candidates, path)
 
 
+def get_services(config, version):
+    if version == 2:
+        return config.get('services', {})
+    else:
+        return config
+
+
+@validate_top_level_object
+def get_config_version(config):
+    return config.get('version')
```

Comment:
This doesn't need to be in a separate function. It's a dict lookup so can just be a line in `pre_process_config` function and then that way we're not doing `validate_top_level_object` twice.